backend/vkt/src/main/java/fi/oph/vkt/model/type/EnrollmentStatus.java

```diff
@@ -3,6 +3,8 @@
 public enum EnrollmentStatus {
   PAID,
   EXPECTING_PAYMENT,
+  EXPECTING_PAYMENT_PUBLIC,
```

Comment:
Olisko parempi jos nimeäisi nämä uudet tilat vaikkapa `EXPECTING_PAYMENT_UNFINISHED_ENROLLMENT` ja samoin `CANCELED_UNFINISHED_ENROLLMENT` ? Tai unfinished sanan tilalle incomplete, kummin vaan. Mut erottuisi se ilmoittautumisen keskeneräisyys nimestä.

Reply:
Virkailijan UI ei taida muuten näitä erityisemmin parhaillaan hanskata vai ovatko siellä jo? Mietin pitäiskö näyttää tilaisuuden osalta odottaa maksua taulukossa myös nämä keskeneräiset maksua odottavat ilmoittautumiset.

Reply:
Oon tuossa #484 pullarissa muutellu tuota virkailijan puolta niin voisin siellä katsoa nämä kuntoon sitten kun tämä mergetty.